### PR TITLE
Update messages_en-2.txt

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?177329

Haven't yet figured out why this is affecting Lily's app and not others, but reasonably confident that this line does not belong in this file.

@benrudolph

### CODE REVIEW

File: messages_en-2.txt [hunk 1]
```diff
@@ -453,8 +453,6 @@ verify.title=CommCare Resource Verification
 verify.checking=Verifying Resources
 verify.progress=Verifying resources. ${0} resources verified of ${1} total
 
-modules.m0=pregenancy
```
Comment (czue) [inline messages_en-2.txt]: the spelling is the best part

**Comment by dannyroberts** [comment]
This for some reason breaks this test: https://travis-ci.org/dimagi/commcare-hq/jobs/73260671
```
======================================================================
FAIL: test_no_change (corehq.apps.app_manager.tests.test_bulk_ui_translation.BulkUiTranslation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_bulk_ui_translation.py", line 25, in test_no_change
    dict(translations), dict()
AssertionError: {u'en': {u'modules.m0': u'pregenancy'}} != {}
- {u'en': {u'modules.m0': u'pregenancy'}}
+ {}
----------------------------------------------------------------------
```

**Comment by orangejenny** [comment]
@dannyroberts yes :( but the test fix is PRed :)

**Comment by dannyroberts** [comment]
yay!

**Pull request merged**